### DISCUSSION TITLE
Error handling

### DISCUSSION THREAD

**Discussion by sfiannaca** [Mobile-Artificial-Intelligence/maid#83] [Ideas]
Hi, I found a valuable article on enhancing error handling about "Isolate"

https://medium.com/dartlang/better-isolate-management-with-isolate-run-547ef3d6459b

**Comment by sfiannaca** [comment]
I tried to make some changes, but when stuff goes down in the native code, the app crashes without logs.

**Comment by danemadsen** [comment]
Ill look into it. The way i was thinking of doing it is redirecting stdout from core.cpp into Logger.log() in dart via a callback.
